apis/itemsetapi/itemsetapi.yaml

```diff
@@ -0,0 +1,840 @@
+      security:
+        - bearer: []
+      x-unitTests:
+        - request:
+            method: POST
+            uri: /itemset/v1/create
+            headers:
+              Content-Type: application/json
+              X-Channel-id: channel
+              Authorization: 'Bearer {{api_key}}'
+            body: |-
+              {
+                "request": {
+                  "itemset": {
+                    "name": "Test ItemSet",
+                    "description": "Test ItemSet",
+                    "createdBy": "89cf1a7e-dfd3-46c9-a428-d37e9a2bc001"
+                  }
+                }
+              }
+          expectedResponse:
+            x-allowExtraHeaders: true
+            x-bodyMatchMode: NONE
+            x-arrayOrderedMatching: false
+            x-arrayCheckCount: false
+            x-matchResponseSchema: true
+            headers: {}
+          x-testShouldPass: true
+          x-testEnabled: true
+          x-testName: Create ItemSet
+          x-testDescription: ''
+      x-operation-settings:
+        CollectParameters: false
+        AllowDynamicQueryParameters: false
+        AllowDynamicFormParameters: false
+        IsMultiItemSetStreaming: false
+  /update/{ItemSet_ID}:
+     patch:
+      description: >-
+        This API is associated with updating the item set on the Sunbird Platform.
+          - The endpoint for **Update ItemSet** is `/update/{ItemSet_ID}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+
+      summary: Update ItemSet
+      tags:
+        - ItemSet APIs
+      operationId: Update ItemSet
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: ItemSet_ID
+          type: string
+          required: true
+          description: 'Please append a valid ItemSet ID To the Request URL'   
+        - name: Body
+          in: body
+          required: true
+          description: >- 
+            The body parameter refers to the format of the request, the request body (payload) takes the following media types to make the payload:'
+            
+            - JSON
+          
+            The body is the representation of the resource object for updating the item set.
+          schema:
+            $ref: '#/definitions/UpdateItemSetRequest'
+        - name: Content-Type
+          in: header
+          required: true
+          type: string
+          description: >-
+            The ItemSet Type entity is the media type of the resource.Possible media types can be:- 
+              - application/json
+        - name: X-Authenticated-Userid
+          in: header
+          required: true
+          type: string
+          description: 'Represents the User ID authorized to execute the API.'
+        - name: X-Channel-id
+          in: header
+          required: true
+          type: string
+          description: 'Represents a physical location (e.g: state) uniquely to which the user belongs.'
```

Comment:
do you mean the tenant ID? Why does it need to be a physical location? pls check this for all X-Channel-id parameters in the API